source/libs/utils.js

```diff
@@ -142,9 +142,9 @@ export const flatZip = (table, limit = Infinity) => {
 	return zipped;
 };
 
-export const isMac = /Mac/.test(window.navigator.platform);
+export const isMac = () => /Mac/.test(window.navigator.platform);
```

Comment:
I prefer not having to change code because of testing. I already commented a way to fix this properly.

Reply:
Like I said, the issue itself isn't because of a missing mock for the platform property, but rather that the `window` global isn't initialized when the `utils` file is imported.

```js
import test from 'ava';
import * as pageDetect from '../source/libs/page-detect'; // Imports utils.js, error is thrown because no window global
import Window from './fixtures/window';

// global mocks
global.window = new Window();
global.location = window.location;
global.document = {};
```

I tried creating a new file which sets the `window` global mocks, and is imported before the `page-detect` file. That didn't appear to solve the issue, as the `utils` file is being imported from somewhere else (?)